models/models.js

```diff
@@ -13,7 +13,7 @@ exports.selectArticleById = async (id) => {
     FROM articles AS a
```

Comment:
OOOOOH don't like these being called a & c! Be a little more descriptive, please!